plugins/GigPlayer/CMakeLists.txt

```diff
@@ -6,8 +6,14 @@ if(LMMS_HAVE_GIG)
 	SET(GCC_COVERAGE_COMPILE_FLAGS "-fexceptions")
 	add_definitions(${GCC_COVERAGE_COMPILE_FLAGS})
 
-    LINK_DIRECTORIES(${GIG_LIBRARY_DIRS})
-    LINK_LIBRARIES(${GIG_LIBRARIES})
+	# disable deprecated check for mingw-x-libgig
+	if(LMMS_BUILD_WIN32)
+		SET(GCC_GIG_COMPILE_FLAGS "-Wno-deprecated")
+		add_definitions(${GCC_GIG_COMPILE_FLAGS})
+	endif(LMMS_BUILD_WIN32)
+
+	LINK_DIRECTORIES(${GIG_LIBRARY_DIRS} ${SAMPLERATE_LIBRARY_DIRS})
```

Comment:
@tresf I don't know, seams to work even without, but they were were also in sf2_player, so I just copied it from there.